lms/views/email.py

```diff
@@ -98,4 +102,7 @@ def set_preferences(self):
                 },
             )
         )
+        self.request.session.flash(
+            "Preferences successfully saved.", "email_preferences_result"
```

Comment:
The word "successfully" feels redundant here given how we're presenting the message. "Preferences saved" would be fine.

Reply:
I think "preferences" is also unnecessary, I'd suggest just "Saved"